src/cpp/rtps/security/SecurityManager.cpp

```diff
@@ -701,7 +702,7 @@ bool SecurityManager::discovered_participant(
     {
         // Maybe send request.
         returnedValue = on_process_handshake(participant_data, remote_participant_info,
-                        MessageIdentity(), HandshakeMessageToken());
+                        MessageIdentity(), HandshakeMessageToken(), notify_part_authorized);
     }
 
     restore_discovered_participant_info(participant_data.m_guid, remote_participant_info);
```

Comment:
Nice catch ! forgot to bring the change to the final workspace. The pipe was working anyway because the call in the stateless callback is the first one to notifyAbove(). But, of course, we have to include it here